frontend/src/components/ExpenseClaimSummary.vue

```diff
@@ -84,5 +82,61 @@ const total_claimed_amount = computed(() => {
 	)
 })
 
+const employee = inject("$employee")
+const dayjs = inject("$dayjs")
+
+let selectedPeriod = ref({})
+let periodsByName = ref({})
+
+function getPeriodLabel(period) {
+	return `${dayjs(period?.start_date).format("MMM YYYY")} - ${dayjs(
+		period?.end_date
+	).format("MMM YYYY")}`
+}
+
+const summary = createResource({
+	url: "hrms.api.get_expense_claim_summary_for_payroll_period",
+	makeParams(values) {
+		return {
+			employee: employeeResource.data.name,
+			start_date: selectedPeriod.value.start_date || "",
+			end_date: selectedPeriod.value.end_date || ""
+		}
+
+	},
+	auto: true,
+})
+
+watch(
+	() => selectedPeriod.value,
+	(value) => {
+		summary.reload()
+	}
+)
+
+const payrollPeriods = createListResource({
```

Comment:
IMO, Expense claims should be based on the fiscal year and not the payroll period since its mostly settled using payment entries/journal entries and not via payroll.